### PR TITLE
Minor suggestions

### PR DESCRIPTION
**Pull request by seratch**
This project is fascinating! 👍 

This pull request is a collection of minor suggestions for this project. If you like some of these, please feel free to check-pick only relevant ones!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "dev": "source ./.env && nodemon index.js",
+    "dev": "source ./.env && nodemon --experimental-modules index.js",
```
Comment (seratch) [inline package.json:8]: As a workaround for the following error with node v12.16.1

```
[nodemon] starting `node index.js`
internal/modules/cjs/loader.js:1174
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/ksera/github/workflow-powerups/index.js
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1174:13)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'ERR_REQUIRE_ESM'
}
```

File: package.json [hunk 2]
```diff
@@ -27,6 +27,6 @@
     "redis": "^3.0.2"
   },
   "devDependencies": {
-    "nodemon": "^1.19.4"
+    "nodemon": "^2.0.4"
```
Comment (seratch) [inline package.json:30]: npm audit suggests upgrading this

File: random-string-step/index.js [hunk 3]
```diff
@@ -20,17 +20,16 @@ export const registerRandomStringStep = function (app) {
         strings: get(inputs, "strings.value", []),
       };
 
-      app.logger.info("Opening config view", state);
-      await app.client.views.open({
-        token: context.botToken,
+      logger.info("Opening config view", state);
+      await client.views.open({
```
Comment (seratch) [inline random-string-step/index.js:24]: As `client` in listener args embodies `context.botToken`, you can simplify the code with it.

File: random-string-step/index.js [hunk 4]
```diff
@@ -97,18 +95,18 @@ export const registerRandomStringStep = function (app) {
       params.step_image_url = text5;
     }
 
-    app.logger.info("updateStep params: ", params);
+    logger.info("updateStep params: ", params);
 
     // Call the api to save our step config - we do this prior to the ack of the view_submission
     try {
-      await app.client.apiCall("workflows.updateStep", params);
+      await client.workflows.updateStep(params);
```
Comment (seratch) [inline random-string-step/index.js:102]: `workflows.*` methods are available.

File: random-string-step/index.js [hunk 5]
```diff
@@ -119,21 +117,20 @@ export const registerRandomStringStep = function (app) {
     const values = strings.value || [];
 
     // Grab a random string
-    var randomString = values[Math.floor(Math.random() * values.length)];
+    const randomString = values[Math.floor(Math.random() * values.length)];
```
Comment (seratch) [inline random-string-step/index.js:120]: nit

File: update-slack-status-step/index.js [hunk 6]
```diff
@@ -141,7 +138,7 @@ export const registerUpdateSlackStatusStep = function (app, storage) {
   app.action("connect_account_button", async ({ ack }) => ack());
 
   // Handle saving of step config
-  app.view(VIEW_CALLBACK_ID, async ({ ack, view, body, context }) => {
+  app.view(VIEW_CALLBACK_ID, async ({ ack, view, body, logger, client }) => {
     // Pull out any values from our view's state that we need that aren't part of the view submission
     const { userName, userImage } = parseStateFromView(view);
```
Comment (seratch) [inline update-slack-status-step/index.js:143]: nit: `userImage` here seems to be unused

File: random-string-step/index.js (hunk 3 above)
Comment (mbrevoort) [reply random-string-step/index.js:24]: interesting, I did not know that :)

**Review by mbrevoort — APPROVED** [review]
Thanks @seratch! +1

**Pull request merged**